content/auto_pinned_posts.json

```diff
@@ -0,0 +1,6 @@
+[
+	{
+		"text": "mercatino digitale",
+		"user": ["automoderator"]
```

Comment:
Piccola nota più di comodità di lettura nostra, visto che è un array rinomina `"user"` in `"users"`